src/libpmemfile/preload.c

```diff
@@ -229,6 +231,40 @@ log_write(const char *fmt, ...)
 	syscall_no_intercept(SYS_write, log_fd, buf, len);
 }
 
+void
+exit_group_no_intercept(int ret, const char *msg)
+{
+	if (msg) {
+		if (msg[0] == '!') {
+			syscall_no_intercept(SYS_write, 2, msg + 1,
```

Comment:
Oh well, I guess, since we use libc here anyways (`strerror_r`, `snprint_f`, etc...) we could call libc `fputs` here, or libc `write`.
It is advised (and often required) to circumnavigate libc while coding in libsyscall_intercept, but that is not so important here.
libc would call libpmemfile again (when doing a write syscall), but the reenter flag should trivially take care of that.